.gitignore

```diff
@@ -1,3 +1,3 @@
-vspd
+*.exe
```

Comment:
No issue with adding `*.exe`, but why remove `vspd`?

Reply:
vspd don't generate folder `vspd`? I assumed it was intended to ignore `vspd.exe` file.

Reply:
On Linux the executable is not named `vspd.exe`, is named just `vspd` without a file extension.

Reply:
Thanks for your feedback, very helpful.